Source/WebCore/testing/Internals.cpp

```diff
@@ -4214,7 +4214,7 @@ Ref<ArrayBuffer> Internals::serializeObject(const RefPtr<SerializedScriptValue>&
 
 Ref<SerializedScriptValue> Internals::deserializeBuffer(ArrayBuffer& buffer) const
 {
-    return SerializedScriptValue::createFromWireBytes(buffer.toVector());
+    return SerializedScriptValue::createFromWireBytes(buffer.vector());
```

Comment:
Seems like the `SerializedScriptValue::createFromWireBytes` function could take a `std::span` instead of a `Vector`.